### PR TITLE
OTT-784: As a Header bidding module, I should be able to update wrapper logger object with rejected bids (both PSB and HB side) so that analytics team can consume it

### PR DESCRIPTION
**Pull request by PubMatic-OpenWrap**
# Description

Please add change description or link to ticket, docs, etc.

# Checklist:

- [ ] PR commit list is unique (rebase/pull with the origin branch to keep master clean).
- [ ] JIRA number is added in the PR title and the commit message.
- [ ] Updated the `header-bidding` repo with appropiate commit id.
- [ ] Documented the new changes.

For Prebid upgrade, refer: https://inside.pubmatic.com:8443/confluence/display/Products/Prebid-server+upgrade

### CODE REVIEW

File: exchange/exchange.go [hunk 1]
```diff
@@ -983,8 +983,8 @@ func applyCategoryMapping(ctx context.Context, r *AuctionRequest, requestExt *op
 					if r.LoggableObject != nil {
 						r.LoggableObject.RejectedBids = append(r.LoggableObject.RejectedBids, analytics.RejectedBid{
 							Bid:             bid.bid,
-							RejectionReason: openrtb3.LossCategoryExclusions,
-							Seat:            seatBid.seat,
+							RejectionReason: openrtb3.LossBidCategoryExclusions,
```
Comment (ashishshinde-pubm) [inline exchange/exchange.go:986]: Ideally this should be LossBidCategoryMapping

Comment (ShriprasadM) [reply exchange/exchange.go:986]: have we checked @ashishshinde-pubm's comment?

File: openrtb_ext/response.go [hunk 2]
```diff
@@ -99,9 +99,9 @@ type SeatNonBid struct {
 
 // NonBid defines the contract for bidresponse.ext.debug.seatnonbid.nonbid
 type NonBid struct {
-	ImpId      string           `json:"impid,omitempty"`
-	StatusCode NonBidStatusCode `json:"statuscode,omitempty"`
-	Ext        *ExtNonBid       `json:"ext,omitempty"`
+	ImpId      string                    `json:"impid,omitempty"`
+	StatusCode openrtb3.NonBidStatusCode `json:"statuscode,omitempty"`
```
Comment (ShriprasadM) [inline openrtb_ext/response.go:103]: in PR on PBS side, I have kept `StatusCode` as `int`. But here we are keeping `openrtb3.NonBidStatusCode`. Let it be for now. In future we can make it inline with PBS code

**Review by ShriprasadM — CHANGES_REQUESTED** [review]
Please check Review comments

**Pull request merged**